strax/chunk.py

```diff
@@ -31,7 +34,8 @@ def __init__(self,
                  run_id,
                  start,
                  end,
-                 data):
+                 data,
+                 target_size=default_chunk_size):
```

Comment:
Can we be smart about the naming such that no-one accidentally assumes that the target_size is in MB (which would result in saying target_size=200 which would then result in infinite files in the ./strax_data directory where one tests a plugin prior to pushing it to github).

For example, how about target_size_mb (and multiply later by 1000).

Reply:
We could, or we explicitly document it as bytes. We should not be held accountable if someone decides that they don't need to familiarize themselves with the documentation before changing attributes.

Reply:
Agree, people should read the documentation. However, all numbers that can be used here will be in x MB so I think having it in some int between 10-3000 would make it quite readable and easier to spot if someone is doing something that he/she shouldn't. Another example is if you cat the metadata. Having a 1000 vs 1000000000 is somewhat easier to  read.

Reply:
Valid point